server/src/test/java/org/opensearch/action/admin/indices/forcemerge/ForceMergeRequestTests.java

```diff
@@ -109,9 +109,11 @@ public void testBwcSerialization() throws Exception {
                 assertEquals(sample.maxNumSegments(), deserializedRequest.maxNumSegments());
                 assertEquals(sample.onlyExpungeDeletes(), deserializedRequest.onlyExpungeDeletes());
                 assertEquals(sample.flush(), deserializedRequest.flush());
-                if (compatibleVersion.onOrAfter(Version.V_3_0_0)) {
+                if (compatibleVersion.onOrAfter(Version.V_2_13_0)) {
```

Comment:
@gaobinlong why do we need version check in tests? these test are always run against the current version (branch)

Reply:
The version check for the field `primary_only` is not needed actually, I've removed that, thanks! But for the field `forceMergeUUID`, version check is still needed because the serialization for this field is different in 2.x and 3.0.